backend/internal/models/user.go

```diff
@@ -1,12 +1,16 @@
 package models
 
+type UserWithRole struct {
+	User
+	Role ClassroomRole `json:"classroom_role"`
+}
+
 type User struct {
-	ID             int64  `json:"id,omitempty"`
+	ID             *int64 `json:"id,omitempty"`
```

Comment:
I am confused about why we use such large integers for these values

Reply:
IMO going around and using the minimum possible size for every integer would lead to more problems than it's worth right now. That would require us to use differently sized ints for certain IDs that come from external sources such as Github and might involve converting int sizes as well.